### PR TITLE
Adds SS loading to title screen

### PR DESCRIPTION
**Pull request by AffectedArc07**
## What Does This PR Do
Adds SS loading to the title screen - shows visual indicators of whats loading, as well as clearing when all SSs are loaded

## Why It's Good For The Game
Why not

## Images of changes
https://github.com/ParadiseSS13/Paradise/assets/25063394/8db6355b-6734-4e62-b36b-9896cca687ee

## Testing
Started the server

## Changelog
:cl: AffectedArc07
add: Added SS loading stats to lobby screen
/:cl:

### CODE REVIEW

File: code/controllers/master.dm [hunk 1]
```diff
@@ -675,3 +695,79 @@ GLOBAL_REAL(Master, /datum/controller/master) = new
...
+	var/i = 0 // We dont iterate the below loop on i because we only care about initters
+	for(var/datum/controller/subsystem/SS in ss_in_init_order)
+		if(SS.flags & SS_NO_INIT)
+			continue
+
+		i++
+
+		var/list/target_col = columns[length(columns)] // Get most recent once
+
+		if(ceil(i / (max_per_col * length(columns))) > 1)
```
Comment (AffectedArc07) [inline code/controllers/master.dm:723]: SDMM thinks `ceil` doesnt exist.

Comment (S34NW) [reply code/controllers/master.dm:723]: It's not exactly a new proc, did they just forget to add it?

Comment (AffectedArc07) [reply code/controllers/master.dm:723]: Turns out its a 515 thing

Comment (S34NW) [reply code/controllers/master.dm:723]: Oh, I assumed it was like `round` and had been here forever

Comment (AffectedArc07) [reply code/controllers/master.dm:723]: image

**Comment by AffectedArc07** [comment]
- Needs new SDMM release to pass CI unless I make it scroll
- Gonna make it scroll

**Comment by AffectedArc07** [comment]
It scroll now and doesnt need 515 dreamchecker.

**Review by S34NW — APPROVED** [review]

**Pull request merged**